.github/workflows/database-backup.yml

```diff
@@ -23,6 +29,7 @@ jobs:
         shell: bash
         run: |
           tf_vars_file=terraform/paas/workspace_variables/production.tfvars.json
+          echo "namespace=$(jq -r '.namespace' ${tf_vars_file})" >> $GITHUB_ENV
```

Comment:
it seems this is backing up paas db but confirm if we still need to backup paas db going forward